locale/en/invitation.po

```diff
@@ -35,7 +35,7 @@ msgid "userInvitation.enterDetails.stepDescription"
 msgstr "You can invite them to take up a role in OJS"
 
 msgid "userInvitation.modal.message"
-msgstr "{$email} has been invited to new role in OJS.You can be updated about users on the User and Roles page, your OJS notification and/or your email"
+msgstr "{$email} has been invited to new role in OJS. You can be updated about user's on the User and Role page, your OJS notification and/or your email"
```

Comment:
@withanage I'm seeing in the mock ups
"You can be updated about the user's decision on the User and Role page," and changes have
"You can be updated about user's on the User and Role page,"

Reply:
@Tribunal33 
thanks! i have updated the changes.